src/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -235,6 +235,8 @@ def create(dataset, target, feature=None, model = 'resnet-50',
     _tkutl._check_categorical_option_type('model', model, allowed_models)
 
     # Check dataset parameter
+    if not isinstance(dataset, _tc.SFrame):
+        raise _ToolkitError("Unrecognized value for 'dataset'. An SFrame is expected.")
```

Comment:
I think you should say "type" rather than "value" in the error message.

Also I think it makes more sense to raise a `TypeError` rather than a `_ToolkitError`.